ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Improve network performance with Route Reflectors
+description: Learn how to deploy Calico Route Reflectors to improve performance
+  of Kubernetes networking
+keywords: cluster, node, label, certificate, SAN
+---
+
+UCP uses Calico as the default Kubernetes networking solution. Calico is
+configured to create a BGP mesh between all nodes in the cluster.
+
+As you add more nodes to the cluster, networking performance starts decreasing.
+If your cluster has more than 100 nodes, you should reconfigure Calico to use
+Route Reflectors instead of a node-to-node mesh.
+
+This article guides you in deploying Calico Route Reflectors in a UCP cluster.
+UCP running on Microsoft Azure uses Azure SDN instead of Calico for networking.
```

Comment:
UCP running on Microsoft Azure uses Azure SDN instead of Calico for **multi host networking**